Error: Module not found: Error: Cannot resolve 'file' or 'directory' ./css/build/app.css

Hi,
Newbie to these technologies. Could anyone please help me to understand this issue.

Steps i am followed:
1. Git clone the open.fda.gov
2. npm run dev:site (npm run dev is throwing error : npm ERR! missing script: dev)

  Error: Module not found: Error: Cannot resolve 'file' or 'directory' ./css/build/app.css in <directory to open.fda.gov>  yjs/open.fda.gov/src
  resolve file
    ./open.fda.gov/src/css/build/app.css doesn't exist
    ./open.fda.gov/src/css/build/app.css.js doesn't exist
    ./open.fda.gov/src/css/build/app.css.jsx doesn't exist
  resolve directory
    ./open.fda.gov/src/css/build/app.css doesn't exist (directory default file)
    ./open.fda.gov/src/css/build/app.css/package.json doesn't exist (directory description f  ile)
  ./open.fda.gov/src/css/build/app.css]
  ./open.fda.gov/src/css/build/app.css.js]
  ./open.fda.gov/src/css/build/app.css.jsx]
   @ ./src/html.jsx 74:18-53

Thanks in advance

--- a/README.md
+++ b/README.md
@@ -10,7 +10,11 @@ This repository contains the main open.fda.gov website.
 
 ## Running the site
 
-`npm run dev`
+`mkdir -p './src/css/build'`
+
+`npm run dev:css`
+
+`npm run dev:site`
 
 ## Running the site in Docker
 

